gemd/entity/template/process_template.py

```diff
@@ -27,6 +27,10 @@ class ProcessTemplate(BaseTemplate, HasConditionTemplates, HasParameterTemplates
         `Tags <https://citrineinformatics.github.io/gemd-documentation/specification/tags/>`_
         are hierarchical strings that store information about an entity. They can be used
         for filtering and discoverability.
+    allowed_names: List[str], optional
+        The set of names that a process' ingredients are allowed to use in their name field.
+    allowed_labels: List[str], optional
+        The set of labels that a process' ingredients are allowed to use in their labels field.
```

Comment:
A little redundant (names => name field), but it matches the gemd-docs, so I'm not really bothered.